cmake/libprotoc.cmake

```diff
@@ -1,6 +1,4 @@
-set(libprotoc_files
-  ${protobuf_source_dir}/src/google/protobuf/compiler/code_generator.cc
-  ${protobuf_source_dir}/src/google/protobuf/compiler/command_line_interface.cc
+set(libprotoc_files_cpp
```

Comment:
This file is partially auto-generated from [src/Makefile.am](https://github.com/google/protobuf/blob/master/src/Makefile.am#L298). See this script: 
https://github.com/google/protobuf/blob/master/update_file_lists.sh

Please update these files as well:
https://github.com/google/protobuf/blob/master/BUILD
https://github.com/google/protobuf/blob/master/src/Makefile.am
https://github.com/google/protobuf/blob/master/update_file_lists.sh